app/src/main/res/drawable/ic_walking_outside.xml

```diff
@@ -1,12 +1,52 @@
-    <path android:fillColor="#FF000000" android:pathData="M-2.1,265.6V140.4c0,-5 -4.1,-9.1 -9.1,-9.1h0c-5,0 -9.1,4.1 -9.1,9.1v125.2c0,5 4.1,9.1 9.1,9.1h0C-6.2,274.7 -2.1,270.6 -2.1,265.6z"/>
-    <path android:fillColor="#FF000000" android:pathData="M156.9,265.6V140.4c0,-5 -4.1,-9.1 -9.1,-9.1h0c-5,0 -9.1,4.1 -9.1,9.1v125.2c0,5 4.1,9.1 9.1,9.1h0C152.8,274.7 156.9,270.6 156.9,265.6z"/>
-    <path android:fillColor="#FF000000" android:pathData="M-11.2,275.2h159c5,0 9.1,-4.1 9.1,-9.1v0c0,-5 -4.1,-9.1 -9.1,-9.1h-159c-5,0 -9.1,4.1 -9.1,9.1v0C-20.4,271.1 -16.2,275.2 -11.2,275.2z"/>
-    <path android:fillColor="#FF000000" android:pathData="M243,122.9c-16,0 -16,25 0,25S259,122.9 243,122.9z"/>
-    <path android:fillColor="#FF000000" android:pathData="M282.1,185.5c-3.2,-1.8 -6.4,-3.7 -9.8,-5.3c-2.6,-1.3 -7.3,-2.1 -9.4,-4.1c-2.2,-2 -3.4,-6.2 -4.7,-8.9c-0.8,-1.7 -1.6,-3.4 -2.4,-5.1c-0.3,-0.9 -0.7,-1.8 -1.2,-2.7c0,-0.1 0,-0.3 -0.1,-0.4c-0.1,-0.6 -0.3,-1 -0.6,-1.5c-0.2,-0.6 -0.5,-1.2 -0.9,-1.8c-0.6,-1.4 -1.8,-2.5 -3.1,-3c-0.2,-0.1 -0.4,-0.2 -0.6,-0.2c-0.5,-0.3 -0.9,-0.5 -1.4,-0.7c-1.5,-0.6 -2.9,-0.7 -4.3,-0.6c-3,-0.4 -6.4,0.1 -9,1.7c-4.7,2.1 -8.8,5.9 -12.5,9.4c-3.1,2 -6.2,3.9 -9.1,6.2c-7,5.5 -7.8,16.3 -9.9,24.4c-1.6,6.2 8,8.9 9.6,2.7c1.1,-4.1 2.1,-8.2 3.2,-12.3c1.2,-4.5 2.3,-6.4 6.1,-8.9c1.1,-0.7 2.2,-1.5 3.3,-2.3c0.2,0 0.4,-0.1 0.6,-0.2c-0.3,1.2 -0.5,2.5 -0.7,3.8c-0.7,4.8 -1.3,9.6 -2.6,14.3c-0.8,2.5 -1.7,5 -2.1,7.7c-0.6,4.1 -1,8.3 -1.8,12.3c-1.5,4.1 -3.4,8 -5.5,11.4c-4.2,6.9 -9,14.1 -14.9,19.8c-7,6.7 3.6,17.3 10.6,10.6c7.1,-6.8 13.2,-15.7 18.1,-24.3c3.3,-5.7 6,-12.2 7.7,-18.9c1.1,-0.1 2.1,-0.3 3.1,-0.7c1.8,1.7 3.7,3.4 5.5,5.2c3,2.9 7.5,5.8 9.3,9.7c1.9,4.1 2.7,9.3 3.7,13.7c1.1,5 2,10 2.8,15.1c0.6,4 5.7,6.2 9.2,5.2c4.3,-1.2 5.8,-5.2 5.2,-9.2c-1.5,-10.5 -3.1,-22.8 -7.9,-32.3c-2.2,-4.3 -6.1,-7.5 -9.5,-10.8c-3,-2.9 -6.1,-5.7 -9.1,-8.6c1.8,-7 2,-14.3 3.8,-21.2c1.4,3 2.7,6.3 5.1,8.6c2.4,2.4 5.7,3.2 8.7,4.4c4.3,1.8 8.4,4.2 12.4,6.5C282.6,197.3 287.7,188.7 282.1,185.5z"/>
-    <path android:fillColor="#FF000000" android:pathData="M192.4,131.8L76.5,50.7c-0.4,-0.4 -0.8,-0.8 -1.3,-1.2c-0.1,0 -0.1,-0.1 -0.2,-0.1C72,46 66.8,45.3 63,48l-117.2,82.1c-4.1,2.9 -5.1,8.6 -2.2,12.7h0c2.9,4.1 8.6,5.1 12.7,2.2L67.8,66.9l114,79.8c4.1,2.9 9.8,1.9 12.7,-2.2S196.5,134.6 192.4,131.8z"/>
-</vector>
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="300dp"
+        android:height="300dp"
+        android:viewportWidth="300"
+        android:viewportHeight="300">
+
+    <path
+        android:fillColor="#231E20"
+        android:pathData="M233,131c-14.7,0-14.7,22.9,0,22.9S247.7,131,233,131z" />
+    <path
+        android:fillColor="#231E20"
+        android:pathData="M265.9,187.8c-2.8-1.6-5.git s6-3.2-8.5-4.6c-2.3-1.1-6.3-1.8-8.2-3.5c-1.9-1.8-2.9-5.4-4-7.7
```

Comment:
This line has the string "git s" between "6-5." and "6-3.2".  Removing "git s" prevents the crash on the tablet